roles/pem/server/config/webserver/tasks/main.yml

```diff
@@ -84,6 +84,11 @@
 - name: Configure SELinux security policy for PEM
   command: "{{ pem_base_dir }}/bin/configure-selinux.sh"
 
+- name: Ensure web server is started on boot
+  service:
+    name: "{{ pem_web_server_service_name[ansible_os_family] }}"
+    enabled: yes
+
```

Comment:
I can't seem to add a comment for lines that didn't change: 
- down below we always restart pem webserver without any condition ? should we change that ?
- there is a typo in the task right after the restart one, maybe we could fix it while we are at it.